values-secret.yaml.template

```diff
@@ -12,17 +13,35 @@
 #    valuesDirectoryURL: https://github.com/beekhof/patterns/raw/main
   
 secrets:
-  # NEVER COMMIT THESE VALUES TO GIT
-  imageregistry:
+  - name: imageregistry
+    fields:
     # eg. Quay -> Robot Accounts -> Robot Login
-    username: PLAINTEXT
-    password:   PLAINTEXT
+    - name: username
+      onMissingValue: prompt
+      value: null
+      prompt: "Insert Quay Username"
+    - name: password
+      onMissingValue: prompt
+      value: null
+      prompt: "Insert Quay Password"
 
-  git:
+  - name: git
+    fields:
     # Go to: https://github.com/settings/tokens
-    username: PLAINTEXT
-    password: PLAINTEXT
+    - name: username
+      onMissingValue: prompt
+      value: null
+      prompt: "Insert Github Username"
+    - name: password
+      onMissingValue: prompt
+      value: null
+      prompt: "Insert Github Password"
 
-  aws:
-    # Secret used for S3 functionality in manuela-data-lake
-    s3Secret: BASE64STRING
+  - name: aws
+    fields:
+    - name: aws_access_key_id
+      ini_file: ~/.aws/credentials
```

Comment:
If the file doesn't exist, will it prompt for a different location?

Reply:
ATM no, we just error out. Maybe ading some additional comments and/or an alternative commented out section would cover this somewhat? If we want to add a prompt, I think I'd like to do it only when sombody specifies something like "promptIfMissing: true" or the likes (i.e. not automatically)